### PR TITLE
1.18.2 branch

### PR DESCRIPTION
**Pull request by Christofmeg**
Created branch from ([#346](https://github.com/way2muchnoise/JustEnoughResources/tree/95dc8c5afcd9828d14c3c2a62b6abca243e167b3))
Added spawn egg slot to 1.18.2
Changed "Experience Dropped" to "Experience". Ender dragon exp was overlapping with spawn egg slot.
Updated pack_format to match 1.18.2
Updated version to 14.2.1
Wandering Trades
Deepslate Ores
Merged changes from:
- ([https://github.com/way2muchnoise/JustEnoughResources/issues/328](https://github.com/way2muchnoise/JustEnoughResources/issues/328))
- ([https://github.com/way2muchnoise/JustEnoughResources/pull/368](https://github.com/way2muchnoise/JustEnoughResources/pull/368))
- ([https://github.com/way2muchnoise/JustEnoughResources/pull/367](https://github.com/way2muchnoise/JustEnoughResources/pull/367))
- ([https://github.com/way2muchnoise/JustEnoughResources/pull/375](https://github.com/way2muchnoise/JustEnoughResources/pull/375))
- ([https://github.com/way2muchnoise/JustEnoughResources/pull/378](https://github.com/way2muchnoise/JustEnoughResources/pull/378))
- ([https://github.com/way2muchnoise/JustEnoughResources/pull/384](https://github.com/way2muchnoise/JustEnoughResources/pull/384))
-  [https://github.com/way2muchnoise/JustEnoughResources/issues/352](https://github.com/way2muchnoise/JustEnoughResources/issues/352)
- [https://github.com/way2muchnoise/JustEnoughResources/pull/366](https://github.com/way2muchnoise/JustEnoughResources/pull/366)
- Optimizations based upon [https://github.com/way2muchnoise/JustEnoughResources/commit/f29c18a9183aa566a327da130f62b91f63b742d9](https://github.com/way2muchnoise/JustEnoughResources/commit/f29c18a9183aa566a327da130f62b91f63b742d9)

### CODE REVIEW

**Review by way2muchnoise — CHANGES_REQUESTED** [review]
I create the actual 1.18.2 target branch, can you please update where needed?

File: gradle.properties [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+version_forge=40.2.0
 version_minecraft=1.18.2
-version_mapping=2022.03.13-1.18.2
+version_mapping=2022.11.06-1.18.2
 channel_mapping=parchment
 version_major=0
 version_minor=14
-version_revis=1
-version_jei=9.5.3.143
+version_revis=2.1
```
Comment (way2muchnoise) [inline gradle.properties:7]: This should just be 2 and not 2.1

**Review by way2muchnoise — CHANGES_REQUESTED** [review]
one small change remains and we can pull this in

**Comment by way2muchnoise** [comment]
Looks like the build is failing on Travis, does it build locally?

**Comment by way2muchnoise** [comment]
Thanks for the PR

**Comment by Christofmeg** [comment]
@way2muchnoise Will you be uploading the new version to CurseForge / Modrinth in the near future?

**Comment by way2muchnoise** [comment]
Already on CF https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer/files/4445746

Need to look into uploading to Modrinth, had a similar request on [BetterAdvancements#122](https://github.com/way2muchnoise/BetterAdvancements/issues/122).

**Pull request merged**